pro_tes/__init__.py

```diff
@@ -1,2 +1,3 @@
-__version__ = '0.1.0'
+# __version__ = '0.1.0'
```

Comment:
Please remove the commented line. Generally, please don't leave old code in the codebase. If we want old code we can go back in the commit history. One of the beauties of Git ;-)